jenkins-design-language/src/js/components/ResultItem.jsx

```diff
@@ -9,6 +9,12 @@ import { getGlyphFor } from './status/SvgStatus';
 
 import type { Result } from './status/StatusIndicator';
 
+// Makes sure we only linkify explicit URLs with expected protocols
+function validateURL(value: string) {
+    const explicitURL = /^(http|https|file|ftp|mailto):/i;
```

Comment:
add tel protocol maybe?

Reply:
BO is not going to be nice on mobile devices any time soon, and we can easily add it if somebody asks, IMO. But if that's a general consensus it's easy to add.

Reply:
hrm tel could be interesting... click to call @bwalding and he will fix your build... hehehe